### PR TITLE
(tlg3135).tlg3135.tlg001

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Re #2667 I am merging tlg3135.tlg004 and tlg3135.tlg005 into tlg3135.tlg001 since the Argumenta and Chronologia have never been considered separate works from the Epitome Historiarum. 

@lcerrato I'm not sure if this merge will work so if this file fails I'll go back and make sure my container units and other items are valid. Thanks!

### CODE REVIEW

**Comment by lcerrato** [comment]
@AlisonBabeu 
I think this is moot, but for future reference,—

1) you have two documents with a CTS structure of **chapter/book** and a third with **section**, so that would have to be resolved within the refsDecl and div structure. The chronology can't be one level= section because your refsDecl indicates there are two levels and neither is called section.

2) you have added in the two additional works without creating a new container, which results in the errors of duplicate nodes (passages) as seen in the log report: https://app.travis-ci.com/github/OpenGreekAndLatin/First1KGreek/builds/255517642

So, both the original file and the argumentum have the same structure (book/chapter) and since the argumentum is notations on the original edition, the structure numeration is identical in places. Essentially, you've added a second, duplicate set of passages without a container level superseding these that would differentiate between them.

3) To make this work, you have to a) add a new container level that indicates there are three parts here (=works), and name these; and b) you have to add two levels to the chronology and adjust it so that "section" becomes "chapter", so that you'd have a dummy "book" level (just one book) and then the sections are now chapters. (Or do the alternative with the other two works.)

TL;DR is that adding files to the end of the edition doesn't play nice with the CTS structure because you then do not have unique divs/containers throughout the new file.

**Review by lcerrato — APPROVED** [review]
